### ISSUE TITLE
MEModelRunnable (interval:1)

### ISSUE THREAD

**Issue by Saintg0at** [Ticxo/Model-Engine-Wiki#13]
Hi, this is a heavy load on the server
 ModelEngine::Task: com.ticxo.modelengine.manager.MEModelRunnable (interval:1)
 timings: _https://timings.aikar.co/?id=b6eccb5aa2954b40bd833020cd4e8ca2_
 How to fix it?

**Comment by Snoop1CattZ69** [comment]
Try this in config.yml

for first change both rendering mode to B
```
  Mode:
    Animation: B
    Movement: B
```

then move to experimental and enable async

```
Experimental-Features:
  Use-Async: true
  See-OBB: false
```

after changes record new timings and compare.

**Comment by Saintg0at** [comment]
Thanks, this has been fixed in the new version 2.4.2
